docs/user-guide/station-device-provisioning.md

```diff
@@ -0,0 +1,203 @@
+                "chan_multiSF_1": { "enable": true, "radio": 0, "if": -200000 },
+                "chan_multiSF_2": { "enable": true, "radio": 0, "if": 0 },
+                "chan_multiSF_3": { "enable": true, "radio": 0, "if": 200000 },
+                "chan_multiSF_4": { "enable": true, "radio": 1, "if": -300000 },
+                "chan_multiSF_5": { "enable": true, "radio": 1, "if": -100000 },
+                "chan_multiSF_6": { "enable": true, "radio": 1, "if": 100000 },
+                "chan_multiSF_7": { "enable": true, "radio": 1, "if": 300000 }
+              }
+            ],
+            "nocca": true,
+            "nodc": true,
+            "nodwell": true
+          }
+        }
+        ```
+
+      </details>
+
+   4. A more thorough description of `sx1301_conf` can be found at [The LNS Protocol](https://doc.sm.tc/station/tcproto.html?highlight=sx1301conf#router-config-message) specification.
+
+3. If you want to enable client certificate validation for this device, make sure to define the `properties.desired.clientThumbprint` desired property as an array of strings (each of them being one of the allowed thumbprints for client certificates of this device)
+
+4. If you want to enable CUPS for this device, you will need to:
+
+    1. upload the .bundle credential file to the 'stationcredentials' container in the Azure Function Storage Account created by the Starter Kit template ([Quickstart: Upload, download, and list blobs - Azure portal - Azure Storage | Microsoft Docs](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal#upload-a-block-blob))
+    2. make sure to define the `properties.desired.cups` desired property as follows:
+
+    ```json
+    "cups": {
+        "cupsUri": "https://IP_or_DNS:5002",
+        "tcUri": "wss://IP_or_DNS:5001",
+        "cupsCredCrc": INT,
+        "tcCredCrc": INT,
+        "cupsCredentialUrl": "https://...",
+        "tcCredentialUrl": "https://..."
+    }
+    ```
+
+    with
+
+    **'cupsCredCrc'**: computed as CRC32 checksum calculated over the concatenated credentials files `cups.{trust,cert,key}` (or the .bundle file if certificates were generated with the tool provided in this kit)  
+
+    **'tcCredCrc'**: computed as CRC32 checksum calculated over the concatenated credentials files `tc.{trust,cert,key}` (or the .bundle file if certificates were generated with the tool provided in this kit)  
+
+    **'cupsCredentialUrl'**: should point to the blob in the Azure Function storage account containing the concatenated credentials (i.e.: .bundle file generated with the tool provided in this kit)  
+
+    **'tcCredentialUrl'**: should point to the blob in the Azure Function storage account containing the concatenated credentials (i.e.: .bundle file generated with the tool provided in this kit)  
+
+By saving the configuration per LBS in its device twin, the LBS will be able to successfully connect to the LNS and it can start sending frames.
```

Comment:
would it make sense to also add a complete json for the desired props including CUPS setup?

Reply:
I would avoid that, someone could just blindly copy it without understanding.